docs/source/model-registry.rst

```diff
@@ -277,21 +274,6 @@ To fetch a model version by alias, specify the model alias in the model URI, and
 
 Note that model alias assignments can be updated independently of your production code. If the ``champion`` alias in the snippet above is reassigned to a new model version in the Model Registry, the next execution of this snippet will automatically pick up the new model version. This allows you to decouple model deployments from your inference workloads.
 
-**Fetch the latest model version in a specific stage**
-
-To fetch a model version by stage, simply provide the model stage as part of the model URI, and it will fetch the most recent version of the model in that stage.
-
-.. code-block:: python
-
-    import mlflow.pyfunc
-
-    model_name = "sk-learn-random-forest-reg-model"
-    stage = "Staging"
-
-    model = mlflow.pyfunc.load_model(model_uri=f"models:/{model_name}/{stage}")
-
-    model.predict(data)
-
```

Comment:
We can add a small example code for representing how to set and use model alias here. It's ok to do it in follow-up PR.